MetaMask/metamask-extension
# Add signature request warning modal

## Explanation



This PR will add a new warning popover modal when the user signs a request.

## More Information



Fixes #15932 

## Screenshots/Screencaps



### Before



It is a new modal.

### After



![image](https://user-images.githubusercontent.com/97883527/196390603-08399ba2-d3d9-4f8b-a2bf-f0b015506ae2.png)


## Manual Testing Steps



Open the test dapp https://metamask.github.io/test-dapp/
Under the Eth Sign section, click on "Sign"
Notice there is a new warning popover modal displayed


## Pre-Merge Checklist


### + If there are functional changes:
